extension/parquet/parquet_reader.cpp

```diff
@@ -399,7 +399,6 @@ void ParquetReader::InitializeSchema() {
 	if (file_meta_data->schema.size() < 2) {
 		throw FormatException("Need at least one non-root column in the file");
 	}
-	auto root_reader = CreateReader();
```

Comment:
Can this not stay in the `InitializeSchema`? It seems like we are initializing the `root_reader` only right before calling `InitializeSchema` anyway.

Reply:
That's right, moved it to `InitializeSchema()`. Thanks!